data_acquisition/src/runGEEperTile.py

```diff
@@ -180,37 +220,199 @@
+                                            'mean_Aerosol', 'mean_Blue', 'mean_Green', 'mean_Red', 'mean_Nir', 'mean_Swir1', 'mean_Swir2', 'mean_SurfaceTemp', 'mean_temp_qa', 
+                                            'kurt_SurfaceTemp','prop_clouds','prop_hillShadow',
+                                            'pCount_dswe1', 'pCount_dswe3', 'mean_cloud_dist','system:index']))
+    #Check how many existing tasks are running and take a break of 120 secs if it's >25 
+    maximum_no_of_tasks(10, 120)
+    #Send next task.                                        
+    locs_dataOut_89_D1.start()
+    print('Completed Landsat 8, 9 DSWE 1 stack acquisitions for site location at tile ' + str(tiles))
+  
+  else: print('Not configured to acquire DSWE 1 stack for Landsat 8, 9 for sites.')
+  
+  if '3' in dswe:
+    locs_out_89_D3 = locs_stack_ls89.map(ref_pull_89_DSWE3).flatten()
+    locs_srname_89_D3 = proj+'_point_LS89_C2_SRST_DSWE3_'+str(tiles)+'_v'+str(date.today())
+    locs_dataOut_89_D3 = (ee.batch.Export.table.toDrive(collection = locs_out_89_D3,
+                                            description = locs_srname_89_D3,
+                                            folder = proj_folder,
+                                            fileFormat = 'csv',
+                                            selectors = ['med_Aerosol', 'med_Blue', 'med_Green', 'med_Red', 'med_Nir', 'med_Swir1', 'med_Swir2', 'med_SurfaceTemp', 'med_temp_qa',
+                                            'min_SurfaceTemp', 'min_temp_qa', 'min_cloud_dist',
+                                            'max_SurfaceTemp', 'max_temp_qa',
+                                            'Q1_SurfaceTemp', 
+                                            'Q3_SurfaceTemp', 
+                                            'sd_Aerosol', 'sd_Blue', 'sd_Green', 'sd_Red', 'sd_Nir', 'sd_Swir1', 'sd_Swir2', 'sd_SurfaceTemp',
+                                            'mean_Aerosol', 'mean_Blue', 'mean_Green', 'mean_Red', 'mean_Nir', 'mean_Swir1', 'mean_Swir2', 'mean_SurfaceTemp', 'mean_temp_qa',
+                                            'kurt_SurfaceTemp','prop_clouds','prop_hillShadow',
+                                            'pCount_dswe1', 'pCount_dswe3', 'mean_cloud_dist','system:index']))
+    #Check how many existing tasks are running and take a break of 120 secs if it's >25 
+    maximum_no_of_tasks(10, 120)
+    #Send next task.                                        
+    locs_dataOut_89_D3.start()
+    print('Completed Landsat 8, 9 DSWE 3 stack acquisitions for site location at tile ' + str(tiles))
+    
+  else: print('Not configured to acquire DSWE 3 stack for Landsat 8,9 for sites.')
 
-  print('Completed Landsat 8, 9 stack acquisitions for site location at tile ' + str(tiles))
+else: print('No sites to extract Landsat 8, 9 at tile ' +str(tiles))
+  
+#############################################
+##---- LANDSAT 457 POLYGON ACQUISITION ----##
```

Comment:
start polygon implementation

this is nearly identical to above, but doesn't buffer, so separate workflow needed.